content/docs/command-reference/import.md

```diff
@@ -155,7 +155,7 @@ outs:
 ```
 
 Several of the values above are pulled from the original `.dvc` file
-`model.pkl.dvc` in the external DVC repository. The `url` and `rev_lock`
+`data.xml.dvc` in the external DVC repository. The `url` and `rev_lock`
```

Comment:
Thanks, that wasn't right!

But I think it'll be even less confusing to do this:

```suggestion
 [in the external DVC repository](https://github.com/iterative/example-get-started/blob/master/data/data.xml.dvc). The `url` and `rev_lock`
```